CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# v1.8.0:
+# v1.8.1:
```

Comment:
```suggestion
# v1.8.0/v1.8.1:
```

Let's just copy the format for the 1.6.0 release https://github.com/google/osv-scanner/releases/tag/v1.6.1